.github/workflows/preview.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish Any Commit
+on:
+    pull_request:
```

Comment:
Since we're not going to use the preview for every PR, should we create one for every PR? Or should we use label/comment as a trigger?

Reply:
I would just run on every PR, it won't run without approval anyway, but it can be narrowed down.

Reply:
Here's some example workflows: https://github.com/stackblitz-labs/pkg.pr.new?tab=readme-ov-file#examples